### DISCUSSION TITLE
Not Enough Samples to train

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#830] [General Q&A]
I am getting this error, 
![image](https://user-images.githubusercontent.com/88183889/134181357-c6ff8073-8384-46e8-9b52-c6eeaadc2e99.png)

How can I fix this error ? by changing the code

**Comment by ghost** [comment]
As a result of the above problem I am getting tensorboard like this:
![image](https://user-images.githubusercontent.com/88183889/134192601-e20031a7-2a17-430d-8964-cab21cc4a550.png)

**Comment by thorstenMueller** [comment]
There's a simple fix without chaning any code. You need to record more phrases :satisfied:.
You will not get any good results with 30 recordings. Even with the minimum of 100 recordings i doubt that training will make any sense. Think on more than 12 hours of pure recording.

**Reply by ghost** [reply]
@thorstenMueller sir but in your YouTube channel you have done 4 recordings and it works for you !..

**Reply by ghost** [reply]
> There's a simple fix without chaning any code. You need to record more phrases 😆.
> You will not get any good results with 30 recordings. Even with the minimum of 100 recordings i doubt that training will make any sense. Think on more than 12 hours of pure recording.

Is there any other ways to change the code ? which file should I change to make it work .. as in youtube tutorial  you are recorded 4 audio and u trained it
